package.json

```diff
@@ -250,15 +250,15 @@
     "@metamask/key-tree": "^9.0.0",
     "@metamask/keyring-controller": "^7.0.0",
     "@metamask/logo": "^3.1.1",
-    "@metamask/message-manager": "^7.0.2",
+    "@metamask/message-manager": "^7.3.0",
```

Comment:
hey @legobeat I am not sure if that should be the best course of action. I agree that it is usually a good practice to isolate dependencies updates, but in this case the dep update is only relevant to blockaid. Meaning that updating it by itself does not seem relevant. Also some stuff to consider:
* this is a minor version update. We shouldn't expect any breaking change.
* actual version being used was 7.2.0 (on the lock file we can see that yarn was resolving 7.0.2 to 7.2.0 and all other controllers from core were already relying on 7.2.0)
* message-manager update to 7.3.0 is only about adding blockaid validations to messages (and it was added as an optional property to the original request... you can check the PR [here](https://github.com/MetaMask/core/pull/1541/files)).
* If there's indeed an issue with this dep update, we would like to to fix/revert not just the dep update but also the changes on this PR.

Also, we never intended to update message-managers for everything. We thought about keeping it contained to the `signature-controller`. However there are 2 blockers to that:
1. release process from core -> currently core does not allow release a new controller just to bump the workspace dependencies version. As there are no explicit file changes (because we are using yarn workspace ranges `@message-managers:"workspace:^"`), the [create-release-branch](https://github.com/MetaMask/create-release-branch/blob/main/src/repo.ts#L248) package won't be able to detect changes to the package that we want to bump workspace deps. So the only option is to actually update updated package directly in the clients and have the package manager handling resolutions. But taking this case as an example, how could we update message-managers in the `signature-controller` if message-managers was not being used in the clients? 🤔 
2. even if we were indeed able release signatureController from core and only have that being used, `yarn dedupe` (if we do not run it, the CI would fail) would ensure that there would be no duplication and as the range to be included in the `signature-controller` would overlap with the current range (this is a minor version update and we are locking only to major versions). With that in mind, it means that the only safe way to update any dependency would be to do it in isolation?

btw I am not saying that there's a right or wrong, just that it is an interesting problem.. and I believe that we should define some guidelines on how to deal with the issues mentioned above. For now I would suggest moving this PR as is, while we define best practices on how to update dependencies on the clients (and distinguish between a patch, minor or major updates). @legobeat @jpuri @Gudahtt @mcmire what do you think?

Reply:
This is minor update and I do not expect any breaking changes.